### DISCUSSION TITLE
Porkbun updating API hostname 12/1/2024 - api.porkbun.com

### DISCUSSION THREAD

**Discussion by ahendowski** [qdm12/ddns-updater#838] [Ideas]
Received an email stating Porkbun is changing their API url to api.porkbun.com:

> Hello there,
> 
> The reason you are receiving this email is because you have an API key associated with your account. In order to ensure that any apps or tools you may have that utilize our API, we wanted to let you know about some upcoming critical updates.
> 
> We know these kinds of changes are annoying but the time has come to separate the processing of incoming API commands from our website. In previous documentation the hostname [porkbun.com](http://porkbun.com/) was used for the API but we have recently updated it to [api.porkbun.com](http://api.porkbun.com/) and in the near future will start enforcing its use. Please update the hostname for our API to [api.porkbun.com](http://api.porkbun.com/) as soon as possible.
> 
> 
> CRITICAL UPDATE DETAILS
> 
> Type: API Hostname Change
> 
> Old Value: [porkbun.com](http://porkbun.com/)
> 
> New Value: [api.porkbun.com](http://api.porkbun.com/)
> 
> Deadline: 2024-12-01 00:00:00 UTC
> 
> 
> Please note that after the deadline, API calls made to the old hostname will no longer be allowed. If you have any questions or concerns please contact support.
> 
> 
> 
> Thank you,
> The Porkbun Team

Will this affect the DDNS updater?

internal/provider/providers/porkbun/api.go

I'm still fairly new but it seems like the porkbun api file calls for porkbun.com and will need to be changed to api.porkbun.com

Just want to make sure I'm on the right track with this.  Thanks to anyone who responds.

**Comment by qdm12** [comment]
This is done in the latest image (yesterday). Will do a v2.8.2 release soon.
